keps/sig-node/2000-graceful-node-shutdown/kep.yaml

```diff
@@ -0,0 +1,34 @@
+title: Graceful Node Shutdown
+kep-number: 2000
+authors:
+  - "bobbypage"
+  - "mrunalp"
+owning-sig: sig-node
+status: provisional
+creation-date: 2020-09-21
+
+# The target maturity stage in the current dev cycle for this KEP.
+stage: alpha
+
+# The most recent milestone for which work toward delivery of this KEP has been
+# done. This can be the current (upcoming) milestone, if it is being actively
+# worked on.
+latest-milestone: "v1.19"
```

Comment:
should it be 1.20? (based on field description)

Reply:
Yup, you're correct, fixed.